x-pack/plugin/transform/src/internalClusterTest/java/org/elasticsearch/xpack/transform/persistence/TransformConfigManagerTests.java

```diff
@@ -753,7 +753,7 @@ private static ClusterState createClusterStateWithTransformIndex(String... index
             IndexMetadata.Builder builder = new IndexMetadata.Builder(index).settings(
                 Settings.builder()
                     .put(TransformInternalIndex.settings(Settings.EMPTY))
-                    .put(IndexMetadata.SETTING_INDEX_VERSION_CREATED.getKey(), Version.CURRENT)
+                    .put(IndexMetadata.SETTING_INDEX_VERSION_CREATED.getKey(), TransformConfigVersion.CURRENT)
```

Comment:
This is an index version, not a config version. This file shouldn't be changed in this PR.